src/App.js

```diff
@@ -21,65 +25,26 @@ const firebaseConfig = {
 const app = initializeApp(firebaseConfig);
 const db = getFirestore(app);
 
-const collectionName = "tasks-0";
-const tasksCollection = collection(db, collectionName);
-
-function DeleteDialog(props) {
-  return <div className={"backdrop"}>
-    <div className="modal">
-      Delete all completed items?
-      <div className="alert-buttons">
-        <button className={"alert-button alert-cancel"} type={"button"}
-                onClick={() => props.onClose()}>
-          No
-        </button>
-        <button className={"alert-button alert-ok"} type={"button"}
-                onClick={() => {
-                  props.onOK();
-                  props.onClose();
-                }}>
-          Delete
-        </button>
-      </div>
-    </div>
-  </div>
-}
+const tabCollectionName = "tabs-0";
+const tabsCollection = collection(db, tabCollectionName);
 
 function App(props) {
   const [isShowCompleted, setIsShowCompleted] = useState(true);
-  const [tasks, _setTasks] = useState([]);
-  const [isShowDeleteDialog, setShowDeleteDialog] = useState(false);
-  const [_sortField, setSortField] = useState("updated");
+  const [sortOrder, setSortOrder] = useState("updated desc");
+  const [activeTab, setActiveTab] = useState(0);
+  const [modal, setModal] = useState({show: false})
 
-  const sortField = useRef({});
-  sortField.current = _sortField;
+  const tasksCollection = collection(db, `${tabCollectionName}/${activeTab}/tasks`);
 
-
-  function setTasks(arr) {
-    let sorted = arr.sort((a, b) => b[sortField.current] >= a[sortField.current] ? 1 : -1);
-    if (sortField.current === "text") {sorted.reverse();}
-    _setTasks((prev) => [...sorted]);
-    console.log("sorting by", sortField.current)
-  }
+  const [tabs, tabsLoading, tabsError] = useCollectionData(query(tabsCollection, orderBy("created")));
+  const [tasks, tasksLoading, tasksError] = useCollectionData(query(tasksCollection, orderBy(...sortOrder.split(" "))));
```

Comment:
How do you know that there will be such a collection (when you first run, how do you know there will be an activeTab with id 0?